src/mobu/monkey.py

```diff
@@ -113,13 +113,16 @@ async def _runner(self) -> None:
                 if self.state == MonkeyState.RUNNING:
                     self.state = MonkeyState.ERROR
             if run:
+                self.log.warning("Restarting failed monkey after 60s")
```

Comment:
should 60 be a parameter?

Reply:
Meh, we haven't needed to tune it yet. We can add a parameter pretty easily when we run into a case where we want to change it.